webapp/graphite/render/glyph.py

```diff
@@ -885,12 +885,12 @@ def output(self, fileObj):
         metaData = { }
 
       self.surface.finish()
-      svgData = str(self.surfaceData.getvalue())
+      svgData = self.surfaceData.getvalue()
```

Comment:
instead of adding all the `b` below, you could change this to:

```python
svgData = self.surfaceData.getvalue().decode()
```

That should work on both python2 and python3.

(also in this case, keep the original `.encode('utf-8')` at the end)

Reply:
to demystify a bit, here's a demo of python2 and python3 behavior:

```
>>> import sys
>>> sys.version
'2.7.17 (default, Jan 14 2020, 03:01:26) \n[GCC 4.2.1 Compatible Apple LLVM 11.0.0 (clang-1100.0.33.17)]'
>>> val = b"<svg>foobar</svg>"
>>> str(val)
'<svg>foobar</svg>'
>>> val.decode()
u'<svg>foobar</svg>'
>>> val.decode().replace("foobar", "baz").encode('utf-8')
'<svg>baz</svg>'
>>> '<svg>baz</svg>' == b"<svg>baz</svg>"
True
```

```
>>> import sys
>>> sys.version
'3.7.7 (default, Mar 12 2020, 20:55:24) \n[Clang 11.0.0 (clang-1100.0.33.17)]'
>>> val = b"<svg>foobar</svg>"
>>> str(val)
"b'<svg>foobar</svg>'"
>>> val.decode()
'<svg>foobar</svg>'
>>> val.decode().replace("foobar", "baz").encode('utf-8')
b'<svg>baz</svg>'
```

Reply:
If the input is a byte array and the output has to be a byte array. Then what would be the point of transcoding it to a string at the beginning and then transcoding it back to a byte array at the end?

Reply:
just to avoid adding `b` to all the lines of code below